include/z64message.h

```diff
@@ -236,7 +236,10 @@ typedef struct {
     /* 0xE300 */ s32 msgLength; // original name : "msg_data"
     /* 0xE304 */ u8 msgMode; // original name: "msg_mode"
     /* 0xE305 */ char unk_E305[0x1];
-    /* 0xE306 */ u8 msgBufDecoded[200]; // decoded message buffer, may be smaller than this
+    /* 0xE306 */ union {
+        u8 msgBufDecoded[200];
+        u16 msgBufDecodedWide[100];
+    }; // decoded message buffer, may be smaller than this
```

Comment:
think comment can go honestly, think it seems clear the size is correct, and the name makes it clear that its the decoded buffer. but shrug